File: tournaments/parkingCost/parkingCost.cpp
int parkingCost(std::string timeIn, std::string timeOut) {
  int diff = std::stoi(timeOut) * 60 +
            std::stoi(timeOut.substr(3)) -
            std::stoi(timeIn) * 60 -
            std::stoi(timeIn.substr(3));
  if (diff <= 30) {
    return 0;
  }
  if (diff <= 120) {
    return (diff - 21) / 10;
  }
  return 9 + ((diff - 111) / 10) * 2;
}

File: tournaments/parkingCost/parkingCost.js
function parkingCost(timeIn, timeOut) {
    const timeOutNorm = parseInt(timeOut) * 60 + parseInt(timeOut.substr(3));
    const timeInNorm = parseInt(timeIn) * 60 + parseInt(timeIn.substr(3));
    const d = timeOutNorm - timeInNorm;
    
    if (30 >= d) {
        return 0;
    } else if (120 >= d) {
        return ~~((d - 21) / 10);
    } else {
        return 9 + ~~((d - 111) / 10) * 2;
    }
}
